### PR TITLE
GLS edit of agile-overview completed

### PR DESCRIPTION
**Pull request by nmuller66**

### CODE REVIEW

File: docs/slides/content/agile-overview.md [hunk 1]
```diff
@@ -75,16 +75,17 @@ continuous delivery of valuable software.
 
 
 
-### Exercise - Pocket Sized Principles
-Working in teams:
+### Exercise - Pocket-sized Principles
+Working in teams
 1. On a piece of paper make room for 12 boxes.
-2. For each box, write 2 to 3 words maximum that effectively capture each of the 12 principles.
+<!--TechEdit: this instruction seems odd, "make room for" on a piece of paper isn't something one can actually do. Give the action directly. "On a piece of paper, draw 12 boxes." -->
```
Comment (ckavili) [inline docs/slides/content/agile-overview.md:81]: sure, we can go with your suggestion :) 
I'll merge it after your update then ☘️

File: docs/slides/content/agile-overview.md [hunk 2]
```diff
@@ -75,16 +75,17 @@ continuous delivery of valuable software.
 
 
 
-### Exercise - Pocket Sized Principles
-Working in teams:
+### Exercise - Pocket-sized Principles
+Working in teams
 1. On a piece of paper make room for 12 boxes.
```
Comment (ckavili) [inline docs/slides/content/agile-overview.md:80]:
```suggestion
1. On a piece of paper draw 12 boxes.
```

File: docs/slides/content/agile-overview.md (hunk 1 above)
Comment (ckavili) [inline docs/slides/content/agile-overview.md:81]:
```suggestion
```

**Comment by nmuller66** [comment]
Looks good!

**Pull request merged**